autoblocks/_impl/global_state.py

```diff
@@ -25,10 +29,19 @@ def init() -> None:
         daemon=True,
     )
     background_thread.start()
-
+    for sig in [signal.SIGINT, signal.SIGTERM]:
+        _loop.add_signal_handler(sig, lambda: asyncio.ensure_future(_on_exist_signal()))
     _started = True
 
 
+async def _on_exist_signal() -> None:
+    tasks = [x for x in asyncio.all_tasks(loop=_loop) if x.get_coro().__name__ == "_send_event_unsafe" and not x.done()]
```

Comment:
can we add a test or something that will fail if we rename `_send_event_unsafe` to something else? as-is this is kind of out of sight and out of mind, so it would probably be helpful to have something that makes this relationship clear. so maybe:

`util.py`
```python
# Add long comment explaining why we do this
SEND_EVENT_CORO_NAME = "_send_event_unsafe"
```

in our tests:

```python
def test_tracer_has_send_event_unsafe():
    assert AutoblocksTracer._send_event_unsafe.__name__ == SEND_EVENT_CORO_NAME
```

so that if we come in later and forget we've done this and rename the function, this test will fail and we'll find the comment explaining this past decision